leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -52,7 +52,7 @@
     <activity
         android:name="leakcanary.internal.DisplayLeakActivity"
         android:enabled="false"
-        android:icon="@mipmap/leak_canary_icon"
+        android:icon="@mipmap/ic_launcher"
```

Comment:
All resource names in leakcanary have to start with `leak_canary` (there's a build check for that)